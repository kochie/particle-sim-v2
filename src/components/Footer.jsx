import React from 'react';

export default class Footer extends React.component {
	render() {
		return (
			<div></div>
		)
	}
}