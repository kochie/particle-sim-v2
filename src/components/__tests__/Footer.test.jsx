import renderer from 'react-test-renderer';
import React from 'react';

import Footer from '../Footer';

function init(props) {
  const instance = renderer.create(<Footer {...props} />);
  return { instance };
}

describe('footer react component', () => {
  it('should render and match snapshot', () => {
    const { instance } = init();
    expect(instance).toMatchSnapshot();
  });
});
