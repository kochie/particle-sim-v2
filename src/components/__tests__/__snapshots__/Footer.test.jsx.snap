// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer react component should render and match snapshot 1`] = `<div />`;
